app.js

```diff
@@ -1,31 +1,31 @@
-// const express = require('express');
-// const routes = require("./routes")
-// const fs = require("fs");
+const express = require('express');
+const routes = require("./routes")
+const fs = require("fs");
 
-// const app = express();
-// const port = 3000;
+const app = express();
+const port = 3000;
 
-// app.use(express.json());  // parse body as jason.
-// // app.use(bodyParser.json());
-// app.use("/api", routes);
+app.use(express.json());  // parse body as jason.
+// app.use(bodyParser.json());
+app.use("/api", routes);
 
-// app.route('/')
-//     .get((req, res) => {
-//         res.send("Retrieve an existing resource")
-//     })
-//     .post((req, res) => {
-//         res.send("Create a new resource")
-//     })
-//     .put((req, res) => {
-//         res.send("Update an existing resource")
-//     })
-//     .delete((req, res) => {
-//         res.send("Remove an existing resource")
-//     })
+app.route('/')
```

Comment:
Trying to understand if this `/` endpoints are part of this task? Or shall it be deleted?

Reply:
i think they are valid for nr.3 Task , if not i can remove!

Reply:
Please remove them to keep the PR focused on the task requirements

Reply:
Done

Reply:
I actually thought this twice so I let only a get('/') respond so if by any chance you open the browser at the servers port , you can have a nicely welcoming msg! (what do you think?)

Reply:
It's not a problem for an API. But you can keep it if you like